tests/lib/rules/prop-name-casing.js

```diff
@@ -189,12 +217,156 @@ ruleTester.run('prop-name-casing', rule, {
         }
       `,
       options: ['snake_case'],
+      output: `
+        export default {
+          props: {
+            'greeting_text': String
+          }
+        }
+      `,
       parserOptions,
       errors: [{
         message: 'Prop "greeting-text" is not in snake_case.',
         type: 'Property',
         line: 4
       }]
+    },
+    {
+      filename: 'test.vue',
+      code: `
+        export default {
+          props: {
+            'greeting_text': String
+          }
+        }
+      `,
+      output: `
+        export default {
+          props: {
+            'greetingText': String
+          }
+        }
+      `,
+      parserOptions,
+      errors: [{
+        message: 'Prop "greeting_text" is not in camelCase.',
+        type: 'Property',
+        line: 4
+      }]
+    },
+    {
+      // computed property name
+      filename: 'test.vue',
+      code: `
+        export default {
+          props: {
+            ['greeting-text']: String
+          }
+        }
+      `,
+      output: null,
+      parserOptions,
+      errors: [{
+        message: 'Prop "greeting-text" is not in camelCase.',
+        type: 'Property',
+        line: 4
+      }]
+    },
+    {
+      // shorthand
+      filename: 'test.vue',
+      code: `
+        export default {
+          props: {
+            greeting_text
+          }
+        }
+      `,
+      output: null,
+      parserOptions,
+      errors: [{
+        message: 'Prop "greeting_text" is not in camelCase.',
+        type: 'Property',
+        line: 4
+      }]
+    },
+    {
+      // valiable computed property name
+      filename: 'test.vue',
+      code: `
+        export default {
+          props: {
+            [greeting_text]: String
+          }
+        }
+      `,
+      output: null,
+      parserOptions,
+      errors: [{
+        // bug ?
```

Comment:
I guess so, it shouldn't throw error in this case I think.

Reply:
But we can deal with it in separate task.

Reply:
Thank you for your view
I wrote an issue #442